### DISCUSSION TITLE
[Support] How to allow a user to configure your npm packege

### DISCUSSION THREAD

**Discussion by Errorbot1122** [node-config/node-config#697] [Q&A]
**Note: for support questions, please use [StackOverflow](https://stackoverflow.com/questions/tagged/node-config) and tag your question with `node-config`**. *This repository's issues are reserved for feature requests and bug reports.*

Before submitting a bug report, please search the issue tracker on the wiki first. Many issues have already been discussed.

The wiki is located at: https://github.com/node-config/node-config/wiki

### I'm submitting a ... 
  - [ ] bug report
  - [ ] feature request
  - [X] support request or question => Please do not submit support request or questions here, see note at the top of this template.

### Please tell us about your environment:

 -  node-config version: 3.3.7
 -  node-version: 14.17.1
 
### Other information

(e.g. are you using an environment besides Node.js?)

I'm using [replit](https://replit.com/@ErrorbotTHE2nd/Roblox-File-Handler-1) (CHECK THE UNSTABLE BRANCH)

I can't use **StackOverflow** as I got banned from it a long time ago and I can't do anything 😞 

![Screenshot 2022-03-23 6 16 48 AM](https://user-images.githubusercontent.com/65428515/159837511-7f78a797-dae9-43e6-946a-039f506ede3f.png)

**Comment by markstos** [comment]
I'm sorry you got banned from StackOverflow. We are considering enabling Github Discussions soon. In the meantime, there's no one here volunteering to answer your support request, so I'm closing it.
